backend/plugins/sales_api/src/modules/sales/db/models/Stages.ts

```
@@ -0,0 +1,97 @@
+
+      if (stage) {
+        throw new Error('Code must be unique');
+      }
+    }
+
+    /**
+     * Create a stage
+     */
+    public static async createStage(doc: IStage) {
+      if (doc.code) {
+        await this.checkCodeDuplication(doc.code);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)